pkg/kubecost/kubecost_codecs.go

```diff
@@ -40,10 +40,10 @@ const (
 	CloudCostCodecVersion uint8 = 1
 
 	// DefaultCodecVersion is used for any resources listed in the Default version set
-	DefaultCodecVersion uint8 = 17
+	DefaultCodecVersion uint8 = 19
```

Comment:
It looks like you ran bingen with a 19 flag but did not update the command in the bingen file. The flag corresponds to this block
https://github.com/opencost/opencost/blob/87b4c630de3d99bfc791c8409b565a14be685feb/pkg/kubecost/bingen.go#L23-L26
and I don't think that you changed any of those types here. Increasing the version number is fine but you did not also update this line.
https://github.com/opencost/opencost/blob/87b4c630de3d99bfc791c8409b565a14be685feb/pkg/kubecost/bingen.go#LL86C5-L86C97
which means that next time someone runs bingen the `DefaultCodecVersion` will drop back to version 17. This will cause a parsing failure on all windows. 

I saw this because I just ran bingen. Because I am merging that commit, only a couple of hours after this one I am just reverting it to 17. So no action required.